docs/content/mobile/ios/local-deploy.md

```diff
@@ -1,67 +1,84 @@
-[//]: # (TODO - get rid of mobile_launch.json mention and explain it differently)
+# Connect to a local deployment
 
-# Connecting to a local deployment
+This page describes how to connect an iOS client to a local deployment of SpatialOS. You can do this from inside your [Unity Editor](#in-editor), using the [Unity Remote](#unity-remote) app, using the [iOS Simulator](#ios-simulator) or using your own [iOS device](#ios-device).
```

Comment:
```suggestion
This page describes how to connect an iOS client to a local deployment of SpatialOS. You can do this using either; just your[Unity Editor](#in-editor), the [Unity Remote](#unity-remote) app, the [iOS Simulator](#ios-simulator), or your own [iOS device](#ios-device).
```

This has caught my eye before - it's not entirely clear there are four separate actions - the second and third look like they are part of the first. Tiny rejig. (If I misunderstood, sorry!)